lib/execution_engine2/execution_engine2Impl.py

```diff
@@ -375,6 +375,81 @@ def run_job_batch(self, ctx, params, batch_params):
         # return the results
         return [job_ids]
 
+    def retry_job(self, ctx, params):
```

Comment:
I don't think any of the impl code is tested is it?

Reply:
No, and we don't really have a place to do it except in the load tests file. I added a test that just calls retry_job with some happy path params and checks to see if the call finishes successfully

Reply:
I actually added one when I was debugging some issues that turned out to be in the impl file: https://github.com/kbase/execution_engine2/blob/develop/test/tests_for_integration/api_to_db_test.py